<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DSheet</class>
 <widget class="QWidget" name="DSheet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>入住时间</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_time">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>各种操作</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_id">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>XX详单</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_op">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>331</width>
     <height>181</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>操作</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>持续时间</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>总金额</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_fee">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pay">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>结账</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
